README.md

```diff
@@ -4,67 +4,51 @@
+The creators and maintainers of this list do not receive and should not receive any form of payment to accept a change made by any contributor.
+The goal of this repo is to index open-source projects, not to advertise for profit.
 
-All the links are monitored and tested with [awesome_bot](https://github.com/dkhamsing/awesome_bot) made by  [@dkhamsing](https://github.com/dkhamsing)
+All the links are monitored and tested with [awesome_bot](https://github.com/dkhamsing/awesome_bot) made by [@dkhamsing](https://github.com/dkhamsing)
 
 # MENU
 
 - [What is Docker ?](#what-is-docker-)
 - [Where to start ?](#where-to-start-)
-- [MENU](#menu)
 - [Tools](#tools)
 	- [Continuous Integration / Continuous Delivery](#continuous-integration--continuous-delivery)
 	- [Deployment and Infrastructure](#deployment-and-infrastructure)
-	- [Deployment](#deployment)
-	- [Dev Tools](#dev-tools)
+	- [Developer Tools](#developer-tools)
+	- [Development Environments](#development-environments)
 	- [Docker Compose file](#docker-compose-file)
 	- [Dockerfile](#dockerfile)
-	- [Hosting for containers](#hosting-for-containers)
-	- [Hosting for repositories (registries)](#hosting-for-repositories-registries)
+	- [Garbage Collection](#garbage-collection)
+	- [Hosting Images (registries)](#hosting-images-registries)
+	- [Image Builder](#image-builder)
+	- [Images](#images)
+	- [Linter / Validator](#linter--validator)
 	- [Local Container Manager](#local-container-manager)
-	- [Logging](#logging)
-	- [Metadata](#metadata)
-	- [Monitoring](#monitoring)
+	- [Monitoring & Logging](#monitoring--logging)
 	- [Networking](#networking)
 	- [PaaS](#paas)
 	- [Remote Container Manager / Orchestration](#remote-container-manager--orchestration)
 	- [Reverse Proxy](#reverse-proxy)
 	- [Security](#security)
 	- [Serverless](#serverless)
 	- [Service Discovery](#service-discovery)
-	- [Storing Images and Registries](#storing-images-and-registries)
+	- [Services for running containers](#services-for-running-containers)
 	- [Terminal User Interface](#terminal-user-interface)
-	- [Useful Images](#useful-images)
+	- [Utilities](#utilities)
 	- [Volume management and plugins](#volume-management-and-plugins)
 	- [Web Interface](#web-interface)
 - [Slides](#slides)
-- [Useful Articles](#useful-articles)
-	- [Cloud Infrastructure](#cloud-infrastructure)
-	- [Continuous Integration](#continuous-integration)
-	- [Deep Dive](#deep-dive)
-	- [General Articles](#general-articles)
-	- [Good Tips](#good-tips)
-	- [Main Resources](#main-resources)
-	- [Metal](#metal)
-	- [Multi-Server](#multi-server)
-	- [Networking](#networking)
-	- [Newsletter](#newsletter)
-	- [Optimizing Images](#optimizing-images)
-	- [Other](#other)
-	- [Performances](#performances)
-	- [Raspberry Pi & ARM](#raspberry-pi--arm)
-	- [Security](#security)
-	- [Service Discovery](#service-discovery)
+- [Useful Resources](#useful-resources)
 - [Books](#books)
```

Comment:
I think the books section should also go. Our focus should remain on the tools, etc. Thoughts?